doc/changelogs.md

```diff
@@ -151,3 +151,35 @@ will be most convenient for our users.
 [gfm]: https://guides.github.com/features/mastering-markdown/
 [keep_a_changelog]: https://keepachangelog.com/en/1.0.0/
 [gitlab_changelog]: https://docs.gitlab.com/ee/development/changelog.html
+
+## Alternatives considered
+
+Prior to stick with current process we considered several alternatives. In
+short:
+
+* Collect titles of commits.
+* Collect titles of closed issues.
+* Write changelog entries in a cover letter / pull request and manually move to
+  a release notes draft on GitHub.
+* Single file changelog.
+
+See the previous section for reasons why we didn't pick up something from this
+list.
+
+More alternatives are below.
+
+### Store a changelog entry in a commit message
+
```

Comment:
Okay, looks like you are obsessed with mentioning this useless method. OK, let's add it with mentioned precautions.

Reply:
> Prior to stick with current process we considered several alternatives.

> See the previous section for reasons why we didn't pick up something from this
> list.

> It should mostly work for us, but there are several downsides:

Is it enough?

Reply:
I'd state explicitly: "We'd avoid using this approach due to the following drawbacks:"

Reply:
Applied:

```diff
-This process is [used in librepo][librepo_changelog_process]. It should mostly
-work for us, but there are several downsides:
+This process is [used in librepo][librepo_changelog_process]. We'd avoid using
+this approach due to the following drawbacks:
```